### PR TITLE
Refactor code to remove ignore pylint error comment

### PR DESCRIPTION
**Pull request by insacuri**
https://trello.com/c/3CGwoO1R/2514-remove-ignore-pylint-comments-in-eq-schema-validator

Quick refactor to remove pylint error.
Also added a few more docstrings that pylint was complaining about.  Nothing too informational but it gives hints as to what it returns which can be useful.

### CODE REVIEW

File: app/validation/validator.py [hunk 1]
```diff
@@ -19,8 +19,12 @@ def __init__(self):
         with open('schemas/questionnaire_v1.json', encoding='utf8') as schema_data:
             self.schema = load(schema_data)
 
-    def validate_schema(self, json_to_validate):  # noqa: C901  pylint: disable=too-complex
-
+    def validate_schema(self, json_to_validate):
```
Comment (bitdivision) [inline app/validation/validator.py]: How about: 
```suggestion
    def validate_schema(self, json_to_validate: str) -> List[Dict[str, str]]:
```

Comment (insacuri) [reply app/validation/validator.py]: I agree.... except a change like this I'd want to do over multiple functions in a single PR.
Would a raised issue/trello card be a fair compromise on it?

Comment (bitdivision) [reply app/validation/validator.py]: Fair enough. I was really just testing suggested changes :)

**Comment by MebinAbraham** [comment]
@insacuri Needs squashing.

**Pull request merged**